Adding a CSS interactive example for word-break property

SphinxKnight opened the pull request:
Proposing a simple example for the [word-break](https://developer.mozilla.org/en-US/docs/Web/CSS/word-break) CSS property

SphinxKnight commented:
I've added the index which was missing.

schalkneethling commented on live-examples/css-examples/word-break.html:
```diff
@@ -0,0 +1,31 @@
+<section id="example-choice-list" class="example-choice-list large" data-property="border-radius">
+
```
Nit: You can remove the empty line.

schalkneethling commented on live-examples/css-examples/word-break.html:
```diff
@@ -0,0 +1,31 @@
+        </button>
+    </div>
+
+
```
Nit: You can remove these two empty lines.

schalkneethling commented on live-examples/css-examples/word-break.html:
```diff
@@ -0,0 +1,31 @@
+        次の単語グレートブリテンおよび北アイルランド連合王国で本当に大きな言葉
+      </div>
+    </section>
+</div>
```
Nit: Missing empty line at end of file.

schalkneethling commented on live-examples/css-examples/css/word-break.css:
```diff
@@ -0,0 +1,5 @@
+#example-element {
+ width:10em;
+ text-align: start;
+ border: solid 1px darkgray;
```
Nit: 4 spaces of indentation in CSS

SphinxKnight replied:
Yes, I don't know where those "disappeared". Kinda sorry about that.

SphinxKnight commented:
@schalkneethling thanks a lot for the review, I should have double-checked this kind of stuff, esp. indentation :$ I've also removed the duplicates `initial-choice` attributes and updated the `data-property`.

schalkneethling commented on live-examples/css-examples/css/word-break.css:
```diff
@@ -0,0 +1,5 @@
+    width:10em;
+    text-align: start;
+    border: solid 1px darkgray;
+}
```
Nit: Missing empty last line :-/

SphinxKnight replied:
thanks again, I've finally taken like… 20sec to configure my editor so that it doesn't happen again :)

schalkneethling commented on live-examples/css-examples/css/word-break.css:
```diff
@@ -0,0 +1,5 @@
+#example-element {
+    width:10em;
```
I would suggest making this 80% and also add `padding: 20px` to have some breathing room inside the box.

schalkneethling commented on live-examples/css-examples/word-break.html:
```diff
@@ -0,0 +1,27 @@
+    <section id="default-example" class="default-example">
+      <div id="example-element" class="transition-all">Honorificabilitudinitatibus
+        次の単語グレートブリテンおよび北アイルランド連合王国で本当に大きな言葉
+      </div>
```
Would like to amend this to the following:

```
Honorificabilitudinitatibus califragilisticexpialidocious Pneumonoultramicroscopicsilicovolcanoconiosis
        次の単語グレートブリテンおよび北アイルランド連合王国で本当に大きな言葉
```

SphinxKnight commented:
I've fixed those issues, thanks for taking the time of reviewing.
